### DISCUSSION TITLE
Sudden death should auto-reset map same as Perfect #964

### DISCUSSION THREAD

**Discussion by ziomek64** [ppy/osu#17533] [Ideas] [answered]
What I mean by title is that Sudden death go to game over screen on fail (try again, menu) instead of instantly restarting the map as Perfect mod. It's probably not a bug but seems frustrating, question comes up: why? Let me know what you think. Should be quick fix if it is decided to automatically restart the map. A bit more work is to be done if we add option to settings

**Comment by bdach** [comment, marked as answer]
you should already be able to change this via the mod customization menu, i believe.

**Comment by ziomek64** [comment]
Seems like it, I've just checked. But another issue, if you disable and enable it again it won't save that setting for me.

**Reply by bdach** [reply]
yes. that's how all configurable mods work, i believe. not sure whether that should "save", it highly depends on the mod.

mod presets are eventually planned. that may help here.

**Reply by ziomek64** [reply]
Of course you can't save what mods you have enabled, but customizations behind them should be saved to make it just easier
